### ISSUE TITLE
[Problem] PartDesign: Behavior of the tip property should be changed

### ISSUE THREAD

**Issue by xtemp09** [FreeCAD/FreeCAD#9527]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

0.21 (Development)

### Full version info

```
OS: Windows 10 Version 2009
Word size of FreeCAD: 64-bit
Version: 0.21.0.33110 (Git)
Build type: Release
Branch: master
Hash: 18e40941ceb36c52701a7c74ea26043ef3385701
Python 3.10.10, Qt 5.15.8, Coin 4.0.0, Vtk 9.1.0, OCC 7.6.3
Locale: Russian/Russia (ru_RU)
```

### Subproject(s) affected?

PartDesign

### Issue description

wwmayer [wrote](https://github.com/FreeCAD/FreeCAD/pull/9398#issuecomment-1536452658):

I wonder about the general usefulness of the Move command because it's not even able to handle simple cases.

1. Create a body
2. Create a second body
3. Create a sketch with a rectangle
4. Create a pad
5. Move the pad to the first body. The sketch will be moved too.
6. Modify the pad and confirm your changes
7. You get the warning: `PartDesign::Body: Link(s) to object(s) 'Pad' go out of the allowed scope 'Body001'. Instead, the linked object(s) reside within 'Body Body'`.

The problem is that the second body holds a reference to the moved pad (_Tip_ property). After manually resetting the _Tip_ in the property editor fixes the issue.

### Anything else?

Discussed at #9398

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wwmayer** [comment]
I don't think it's the Tip property to blame but the Move command. It's up to the Move command to make sure that after moving an object to a different body all links of the old body are redirected so that they don't point to objects inside the new body.

**Comment by adrianinsaval** [comment]
I agree, what is missing is that when moving the tip of a body the previous feature should be set as tip in the original body

**Comment by xtemp09** [comment]
@maxwxyz, 0penBrain and wwmayer planned this issue.

**Comment by luzpaz** [comment]
Related:  
* #13016 
* #11132
* #25185
* #24863
* #24922

**Comment by FEA-eng** [comment]
> 5. Move the pad to the first body. The sketch will be moved too.

By dragging in the Tree View? When I try to do that, I get:

`11:46:34  0 Tree.cpp(2559): [ComboView] 'Unnamed#Pad' cannot be dragged out of 'Unnamed#Body001'`

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 26.3.0dev.20260623 (Git shallow)
Build date: 2026/06/23 18:37:30
Build type: Release
Branch: (HEAD detached at 0d88b4ad4)
Hash: 0d88b4ad4e221f05028ac623d15f49f2f2daf626
Python 3.11.14, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, Clipper2 , IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL) [ OS: Polish/Poland (pl_PL) ]
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Navigation Style/Orbit Style/Rotation Mode: Blender/Rounded Arcball/Window center
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```
